astroquery/alma/core.py

```diff
@@ -397,6 +397,8 @@ def query_sia(self, pos=None, band=None, time=None, pol=None,
 
         Parameters
         ----------
+        **kwargs
```

Comment:
would you mind listing the params from above? to format lets you list them without the one-sentence description, and that is still better I think than listing kwargs only (I would rather prefer to remove the whole section that having kwargs only)

Reply:
I tried to fill out the docstring using the same mechanism that is used in `pyvo.dal.sia2.py`, but the documentation build now fails with
```
WARNING: py:obj reference target not found: pyvo.dam.obscore.POLARIZATION_STATES
WARNING: py:obj reference target not found: pyvo.dam.obscore.CALIBRATION_LEVELS
```
I'm not sure if this should be addressed here or if it's `pyvo` documentation that should be updated.

Reply:
I simply removed the offending links from the docstring for now because that seems to be the fastest way of fixing the documentation build without dropping the newly added parameter descriptions. If they don't cause problems for Sphinx at some point in the future then 03238e87e92301b11d327c79f605ad3db815e3fc can be reverted.